### PR TITLE
[Do Not Delete] Fix member finish

### PR DESCRIPTION
**Pull request by TomekFraczek**
Finishing the form sent to members to finish the sign-up process.

This form has two purposes: to have members enter the remainder of the information about themselves, and to force them to actually read the club rules.

Progress:
- [x] Add in the club rules
- [x] Auto-generate member info form section
- [x] Auto-generate membership quiz form section
- [x] Pretty format everything with bootstrap
- [x] Use number widget for phone number
- [x] validate answers (make sure questions are answered correctly)
- [x] Save membership info to database
- [x] Display a success page (maybe the member detail page?)
- [x] (Optional) Make questions be editable via the gui
- [ ] (Optional) Make errors display more neatly. Currently it can be kinda hard to tell which field they belong to
- [ ] (Optional) Require more member info (ie emergency info, address, etc)
- [ ] (Optional) Add in nice background, and club logo

NOTEs: 
- Requires database restart
- Requires a running SMTP server to debug email messages. you can use
```python -m smtpd -n -c DebuggingServer localhost:1024```

### CODE REVIEW

File: core/models/QuizModels.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+class Answer(models.Model):
+
+    answer_text = models.CharField(max_length=100)
+    answer_text.description = "The full answer text to be displayed to the user"
+
+    answer_phrase = models.CharField(max_length=10)
+    answer_phrase.description = "Short one word summary of the answer to be used by the system"
+
+    def __str__(self):
+        return "{phrase}: {text}".format(phrase=self.answer_phrase, text=self.answer_text)
```
Comment (estensen) [inline core/models/QuizModels.py]: Literal String Interpolation is a bit cleaner (Python 3.6+)
`return f'{self.answer_phrase}: {self.answer_text}'`

Comment (estensen) [reply core/models/QuizModels.py]: https://dbader.org/blog/python-string-formatting

**Comment by TomekFraczek** [comment]
I mean, it's only hard-coded in the populate database script, so that questions get generated whenever you restart the database. Once we have a stable version, that script should be deleted.

File: core/models/QuizModels.py (hunk 1 above)
Comment (TomekFraczek) [reply core/models/QuizModels.py]: I keep forgetting this is a thing. Fixed

**Pull request merged**